apache-apisix/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "display_name": "Apache APISIX",
+  "public_title": "Datadog-APISIX Integration",
+  "integration_id": "apache-apisix",
+  "is_public": true,
+  "maintainer": "dev@apisix.apache.org",
+  "manifest_version": "1.0.0",
+  "name": "apache-apisix",
+  "metric_prefix": "apisix.",
+  "metric_to_check": [
+    "apisix.request.counter",
+    "apisix.request.latency",
+    "apisix.upstream.latency",
+    "apisix.apisix.latency",
+    "apisix.ingress.size",
+    "apisix.egress.size"
+  ],
+  "creates_events": false,
+  "short_description": "Datadog-APISIX Integration",
+  "description": "Datadog-APISIX integration using DogStatsD",
+  "guid": "fb69375e-5742-42e6-81cf-8fc19de610ed",
+  "support": "contrib",
+  "supported_os": [
+    "windows",
+    "mac_os",
+    "linux"
+  ],
+  "categories": [
+    "monitoring",
+    "web",
+    "cloud",
+    "log collection",
+    "autodiscovery",
+    "metrics"
+  ],
+  "type": "check",
+  "assets": {
+    "dashboards": {},
+    "configuration": {},
+    "monitors": {},
+    "saved_views": {},
+    "logs": {
+      "source": "apache-apisix"
```

Comment:
I cannot find much documentation about this property 🤔 https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file